torch/csrc/utils/python_dispatch.cpp

```diff
@@ -202,6 +202,15 @@ void initDispatchBindings(PyObject* module) {
     c10::Dispatcher::singleton().checkInvariants();
   });
 
+  m.def("_dispatch_has_kernel_for_dispatch_key", [](const char* name, std::string dispatch) -> bool {
```

Comment:
Does `const std::string&` work for both args?

Reply:
i'll make them both const char*